lock-api/src/main/java/com/palantir/lock/client/LeaderElectionReportingTimelockService.java

```diff
@@ -115,6 +120,24 @@ public ConjureStartTransactionsResponse startTransactions(ConjureStartTransactio
                 .logId());
     }
 
+    public LeaderElectionStatistics statistics() {
+        return statisticsWithRegistry(new DefaultTaggedMetricRegistry());
+    }
+
+    @VisibleForTesting
+    LeaderElectionStatistics statisticsWithRegistry(TaggedMetricRegistry metricRegistry) {
```

Comment:
This is a bit janky, but probably fine for tests. Rename to `getStatisticsAndSetRegistryTo` or something such?

Reply:
Yea, I hate this but it does make it much easier to test.